templates/scripts.tmpl/cleanup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Usage
+# npm run clean
+#
+# To delete node_modules folder:
+# npm run clean -- --include-node-modules
+# OR
+# npm run clean -- --m
+
+rm -rf dist/
+rm -rf dist-node/
+rm -rf dist-browser/
+rm -rf browsertest.build/
+rm -rf .nyc_output/
```

Comment:
I still think it's worth specifying `npm-debug.log` and `lerna-debug.log`, just not a glob for logfiles.